user/serializers.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from django.contrib.auth import get_user_model
+from rest_framework import serializers
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = ("id", "username", "email", "password", "is_staff")
+        read_only_fields = ("id", "is_staff")
+        extra_kwargs = {"password": {"write_only": True, "min_length": 5}}
+
+    def create(self, validated_data):
+        return get_user_model().objects.create_user(**validated_data)
```

Comment:
don't duplicate get_user_model
just declare user_model = get_user_model() in class staticvariables

Reply:
If I declare as you sad it won't work. 
![Untitled](https://user-images.githubusercontent.com/80534232/217942380-04ba5d6a-84f0-414b-9411-197bbacac2ca.png)

Reply:
yeah, It's not so easy as I said before. It's possible but a bit harder
don't care about it this time. let your variant be